scripts/commands/checkPagesRender.ts

```diff
@@ -159,32 +159,27 @@ async function determineFilePaths(args: Arguments): Promise<string[]> {
   }
 
   if (args.nonApi) {
-    globs.push(
-      "docs/support.mdx",
-      "docs/{build,run,start,transpile,verify}/*.{ipynb,md,mdx}",
-      "docs/api/migration-guides/**/*.{ipynb,md,mdx}",
-    );
-  }
-  if (args.currentApis) {
-    globs.push(
-      "docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/*.{ipynb,md,mdx}",
-    );
-  }
-  if (args.historicalApis) {
-    globs.push(
-      "docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/[0-9]*/*.{ipynb,md,mdx}",
-    );
+    globs.push("docs/**/*.{ipynb,md,mdx}");
   }
-  if (args.devApis) {
-    globs.push(
-      "docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/dev/*.{ipynb,md,mdx}",
-    );
-  }
-  if (args.qiskitReleaseNotes) {
-    globs.push("docs/api/qiskit/release-notes/*.{ipynb,md,mdx}");
-  }
-  if (args.translations) {
-    globs.push("translations/**/*.{ipynb,md,mdx}");
+
+  for (const [isIncluded, glob] of [
+    [
+      args.currentApis,
+      "docs/api/{qiskit,qiskit-ibm-provider,qiskit-ibm-runtime}/*.{md,mdx}",
```

Comment:
Note that I removed `.ipynb` from the API docs. We don't use Jupyter notebooks there.